packages/codemirror/src/language.ts

```diff
@@ -473,7 +474,11 @@ export namespace EditorLanguageRegistry {
         filename: /^(BUCK|BUILD)$/,
         async load() {
           const m = await import('@codemirror/lang-python');
-          return m.python();
+          const pythonSupport = m.python();
+          return new LanguageSupport(m.pythonLanguage, [
+            pythonSupport.support,
```

Comment:
Calling `m.python()` should not be need as basically it creates the `LanguageSupport` and we don't care about completion (because JupyterLab brings its own mechanism).

```py
export function python() {
  return new LanguageSupport(pythonLanguage, [
    pythonLanguage.data.of({autocomplete: localCompletionSource}),
    pythonLanguage.data.of({autocomplete: globalCompletion}),
  ])
}
```

Ref https://github.com/codemirror/lang-python/blob/ce4804b2163180989a95d368ffdfa68d48f8d991/src/python.ts#L75

```suggestion
          return new LanguageSupport(m.pythonLanguage, [
```